floater/generators.py

```diff
@@ -76,6 +81,14 @@ def __init__(self, xlim=None, ylim=None, dx=1., dy=1., model_grid=None, load_pat
             self.Ny = int(self.Ly / self.dy)
             self.x = self.xlim[0] + self.dx * np.arange(self.Nx) + self.dx/2
             self.y = self.ylim[0] + self.dy * np.arange(self.Ny) + self.dy/2
+            if zvect is not None:
+                self.zvect = zvect
```

Comment:
It might be safer to cast this explicitly to an array: `self.zvect = np.asarray(zvect)` just to avoid confusion down the line.